post-processing.js

```diff
@@ -229,7 +231,7 @@ function setPasses(rendersettings) {
   composer.addPass(webaverseRenderPass);
   
   if (rendersettings) {
-    const {ssao, dof, hdr, bloom} = rendersettings;
+    const {ssao, dof, hdr, bloom, usePostScene} = rendersettings;

```

Comment:
This can easily be confused with `usePostScene` above but it seems like it's not the same thing.

So can we choose a better name for the config param? `postScene`?

Reply:
Updated usePostScene to enablePostScene in post-processing. The usePostScene hook name in metaversefile-api will stay the same.

Reply:
Updated the usage example comment as well.